Core/Versioning/KspVersionRange.cs

```diff
@@ -27,6 +27,20 @@ public KspVersionRange(KspVersionBound lower, KspVersionBound upper)
             _string = DeriveString(this);
         }
 
+        public KspVersionRange(KspVersion lower, KspVersion upper)
```

Comment:
Could this constructor layer atop `KspVersionRange(KspVersionBound lower, KspVersionBound upper)` ?

```csharp
    : this(lower?.ToVersionRange().Lower, upper?.ToVersionRange().Upper)
```

That way the `DeriveString` logic would be factored out into one common place.